### DISCUSSION TITLE
What to put in `max_tokens` ?

### DISCUSSION THREAD

**Discussion by KaKi87** [zed-industries/zed#22014] [Feature Requests]
Hi,

Zed requires a `max_tokens` property in `language_models.ollama.available_models` items.

But, how to know what to put ?

Thanks

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
